apiary.apib

```diff
@@ -2617,6 +2636,108 @@ Get a list of lost reasons for deals.
                 + id: `6e7fe84d-d4b3-4723-abae-9bc082d8f65a` (string)
                 + name: `Deal declined` (string)
 
+## Deal Pipelines [/dealPipelines]
+
+Each deal belongs to a specific pipeline, in which it goes through the different deal phases.
+
+*Required scopes: `deals`*
+
+### dealPipelines.list [GET /dealPipelines.list]
+
+Get a list of all deal pipelines.
+
++ Request (application/json)
+    + Attributes (object)
+        + filter (object, optional)
+            + `ids`: `46156648-87c6-478d-8aa7-1dc3a00dacab`, `46156648-87c6-478d-8aa7-1dc3a00daca4` (array[string])
+            + status (array, optional)
+                + (enum)
+                    + Members
+                        + open
+                        + pending_deletion
+        + page (Page, optional)
+
++ Response 200 (application/json)
+    + Attributes (object)
+        + data (array)
+            + (object)
+                + id: `f350e48a-fbc3-0a79-e62a-53aa1ae86d44` (string)
+                + name: `Main Pipeline` (string)
+        + meta (object)
+            + `default`: `f350e48a-fbc3-0a79-e62a-53aa1ae86d44` (string)
+            + `page` (Pagination) - Only included with request parameter `includes=pagination`
+            + `matches`: `12` (number) - Only included with request parameter `includes=pagination`
+
+### dealPipelines.create [POST /dealPipelines.create]
+
+Create a new deal pipeline.
+
++ Request (application/json)
+
+    + Attributes (object)
+        + name: `Primary Pipeline` (string, required)
+
++ Response 201 (application/json)
+
+    + Attributes (object)
+        + data (object)
+            + type: `dealPipeline` (string)
+            + id: `f350e48a-fbc3-0a79-e62a-53aa1ae86d44` (string)
+
+### dealPipelines.update [POST /dealPipelines.update]
+
+Update a single deal pipeline.
+
++ Request (application/json)
+
+    + Attributes (object)
+        + id: `254dc5ee-f8de-0791-9e31-a4f7817e64d2` (string, required)
+        + name: `Main Pipeline` (string, required)
+
++ Response 204
+
+### dealPipelines.markAsDefault [POST /dealPipelines.markAsDefault]
```

Comment:
Q: Maybe to soon if we aren't exposing the default on `.list` yet?

Reply:
The `default` pipeline is exposed on the list endpoint as meta data - see: https://github.com/teamleadercrm/api/blob/79a63dab2e8e8d3110f1392394c0b51bade8a4c6/src/03-deals/deal-pipelines.apib#L29